nonlinear_elasticity/nonlinear_elasticity.cc

```diff
@@ -462,23 +462,27 @@ namespace Nonlinear_Elasticity
         n_y = 18;
         n_z = 1;
       }
-    else // FSI3, don't use condition to avoid wmaybe unitialized warning
-      {
-        point_bottom = dim == 3 ? Point<dim>(0.24899, 0.19, -0.005) :
-                                  Point<dim>(0.24899, 0.19);
-        point_tip =
-          dim == 3 ? Point<dim>(0.6, 0.21, 0.005) : Point<dim>(0.6, 0.21);
-
-        // IDs for FSI3/CSM2
-        id_flap_long_bottom  = 2; // x direction
-        id_flap_long_top     = 3;
-        id_flap_short_bottom = 0; // y direction
-        id_flap_short_top    = 1;
-
-        n_x = 25;
-        n_y = 2;
-        n_z = 1;
-      }
```

Comment:
I think you replaced the wrong case. The "case order" of the if statement is not consistent with the linear case but you need to replace the other `if` branch of the PF (not FSI3).